opam

```diff
@@ -1,15 +1,21 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "dave.scott@citrix.com"
+authors: "Dave Scott"
+homepage: "https://github.com/mirage/xentropyd"
+bug-reports: "https://github.com/mirage/xentropyd/issues"
+license: "ISC"
 build: [
   [ "oasis" "setup" ]
   [ "ocaml" "setup.ml" "-configure" ]
   [ "ocaml" "setup.ml" "-build" ]
+  [ "cp" "main.native" "%{bin}%/xentropyd" ]
```

Comment:
my motivation here was to unbreak opam-repo and sync this opam file with opam-repo (based on a build failure of travis in shared-memory-ring) -- xentropyd is currently not used in mirage-entropy anyways -- thus once someone wants to revive it, they can happily do; there seem to be some open issues